script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -53,7 +53,7 @@ Truly――― an unbelievable blue sky.
 More than the white rays of sunlight, it was the harshness of that unblemished blue that pained my eyes.
 <Page15>
 A few days later.
-For a while after I woke up, I couldn't move from the bed.
+For a while, after I woke up, I couldn't move from the bed.
```

Comment:
I don't think this is a correct comma. For the period of time after he first woke up, he couldn't move from the bed. With the comma, it implies that there is a period of time during which he couldn't move from the bed _after waking up_, implying that he could later in the day or something.